packages/db/src/models/base.ts

```diff
@@ -123,13 +123,14 @@ class BasePgModel<ItemType, RecordType extends BaseRecord> {
     return record.cumulus_id;
   }
 
+  // eslint-disable-next-line valid-jsdoc
   /**
    * Get cumulus_id column value for multiple records in Postgres
    *
    * @param {Knex | Knex.Transaction} knexOrTransaction -
    *  DB client or transaction
    * @param {Array<keyof RecordType>} columnNames - column names for whereIn query
-   * @param {Array<string>} values - record values for whereIn query
+   * @param {Array<any>} values - record values for whereIn query
```

Comment:
`any` generally doesn't feel great, I spent a few moments trying to at least limit it to the interface type set for RecordType without anything workable.     Commenting in case that results in a lightbulb moment for a reviewer.